Jenkinsfile

```diff
@@ -13,6 +13,7 @@ pipeline {
    booleanParam(name: 'RUN_DOWNSTREAM', description: 'if false skip downstream jobs', defaultValue: true)
    booleanParam(name: 'RUN_TESTS', description: 'if false skip testing', defaultValue: true)
    booleanParam(name: 'RUN_BUILD', description: 'if false skip building', defaultValue: true)
+   string(name: 'PLATFORMS', description: 'Default Platforms to test', defaultValue: 'nrf9160_pca10090 nrf52_pca10040 nrf52840_pca10056')
```

Comment:
Should we add a few more here? nrf51, nrf52810, etc?

Reply:
With no platforms defined, Sanitycheck runs each testcase once with a random platform.  
When we supply the platform if runs all tests on all platform args. 

If we add every nRF platform the time to run increases dramatically.  
The plan is to add parallel build/test for each platform, and then add nrf51, nrf52810, etc..

Reply:
ok, sounds good